Latest version official

Elaindil opened the pull request.

Elaindil committed:
Latest version official

jerryweb commented:
I think these changes are good, but I also think we may want to allow specific divisions to spawn more of a specific unit than others. For example, thank division can spawn 7 tanks vs other divisions only spawning 4. This would allow us to be able to tell the difference in composition when playing.

Elaindil committed:
Added infantry defenses

added infantry defenses around 12 for defense level 2 and another batch for defense level 3

jerryweb committed:
Added Sandbag script for enemy emplacements

Created sandbag script for enemy emplacements
Added light, medium and heavy at gun tags

jerryweb committed:
Fixed Kalinin and glushovko

jerryweb committed:
Fixed Free Roam script as well as modified the bot for squad order slightly.

Elaindil committed:
Implemented radio trucks without scripts atm and updated the year list

Elaindil committed:
Merge branch 'cookie' of https://github.com/jerryweb/Conquest-Enhanced-Mod into cookie

Elaindil committed:
Added minimap radio truck added side tags

Elaindil committed:
Changes to stuff

Check the tasks page to see changes

Elaindil committed:
Live patch
